cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -226,14 +226,25 @@
   end
   action :nothing
 
+  notifies :run, 'ruby_block[notify sharelib update]', :immediately
+end
+
+ruby_block 'notify sharelib update' do
+  block do
+    node['bcpc']['hadoop']['oozie_hosts'].each do |oozie_host|
+      update_oozie_sharelib(float_host(oozie_host['hostname']))
+    end
+  end
+  action :nothing
+  # Run so that we update the sharelib for this version
   notifies :run, 'ruby_block[oozie sharelib sqoop-action '\
-                 'workaround for 2.6.1]', :immediately
+                 'workaround for 2.6.3]', :immediately
 end
 
-ruby_block 'oozie sharelib sqoop-action workaround for 2.6.1' do
+ruby_block 'oozie sharelib sqoop-action workaround for 2.6.3' do
   block do
     active_release = node['bcpc']['hadoop']['distribution']['active_release']
-    if active_release == '2.6.1.17-1'
+    if active_release == '2.6.3.0-235'
```

Comment:
Why hard code the version strings here? Shouldn't all releases as far as we know need this until they don't?

Reply:
originally i thought it will just be for one release

Reply:
fixed